source/includes/entityref/paybill/_customerrequiredfieldconfiguration.md

```diff
@@ -0,0 +1,154 @@
+# Pay and Bill - CustomerRequiredFieldConfiguration
+
+An entity that represents the primary view for the CRF EDE. CustomerRequiredFieldConfiguration is an EDE entity designed to associate a CustomerRequiredField to a Placement. They have a 1:1 relationship with Placements signifying that only one Placement can have that specific Customer Required Field Configuration.
```

Comment:
You probably want to define each acronym the first time you use it in the doc.

for the Customer Required Field (CRF) Effective Dated Entity (EDE).